packages/sdks/src/components/content-variants/content-variants.lite.tsx

```diff
@@ -37,6 +37,11 @@ type VariantsProviderProps = ContentVariantsPrps & {
    * For internal use only. Do not provide this prop.
    */
   isNestedRender?: boolean;
+
+  /**
+   * Sets the host of Builder API calls. (Defaults to global `https://cdn.builder.io`)
+   */
+  apiHost?: string;
```

Comment:
isn't this a duplicate of the type in `packages/sdks/src/components/content-variants/content-variants.types.ts`? why do we need this one?

Reply:
Yes, good catch! It is not required. Deleting this.